content/en/docs/concepts/overview/kubernetes-api.md

```diff
@@ -104,8 +105,10 @@ are provided in the following format:
             "serverRelativeURL": "/openapi/v3/apis/admissionregistration.k8s.io/v1?hash=E19CC93A116982CE5422FC42B590A8AFAD92CDE9AE4D59B5CAAD568F083AD07946E6CB5817531680BCE6E215C16973CD39003B0425F3477CFD854E89A9DB6597"
         },
         ...
+    }
 }
 ```
+<!-- for editors: intionally use yaml instead of json here, to prevent syntax highlight error. -->
```

Comment:
The snippet is not a valid JSON, and it is not a valid YAML because of the `...` characters.
I'd suggest we use `none` as the syntax here.

Reply:
@tengqm we have quite a few YAML examples that aren't valid YAML but that we highlight as YAML. They seem to work OK.

Reply:
They seem to work okay because ... the YAML parser we use is not strict. Not intended to block this change anyway.